dev-start.sh

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 set -o errexit
-trap "kill 0" EXIT
```

Comment:
I defer to you with bash. You were able to end everything cleanly with a SIGINT?